scripts/start

```diff
@@ -3,8 +3,126 @@
 set -e
```

Comment:
The start script runs npm install twice.
There is an error when running the css watch:
ffc-demo-web-css-watch_1  | npm ERR! enoent ENOENT: no such file or directory, open '/usr/src/app/package.json'
ffc-demo-web-css-watch_1  | npm ERR! enoent This is related to npm not being able to find a file.

Reply:
The app root is mounted into the css-watch container so it should definitely have `package.json` in there. Could be a permission issue - can you check the contents of the app root within the container?
```
docker-compose run ffc-demo-web-css-watch ls -l
```

I can't see any reason for `npm install` running twice for you. I've tested a few times and only ever seen it run once per build. The css-watch and main app containers have identical values for `build` and `image` in `docker-compose.yaml` so Docker should use its build cache for whichever service is built second. The only thing that should cause `npm install` to run twice is if the package.json or package-lock.json is changed while the first image is building.